### DISCUSSION TITLE
Only Glow, no other hacks

### DISCUSSION THREAD

**Discussion by ghost** [danielkrupinski/Osiris#3471] [Q&A]
Hello,

I just want Glow, nothing else...I can I do that before building?

Ty,

Bh

**Comment by diekunstderfuge** [comment]
You could just use Anubis or GoEsp:

https://github.com/danielkrupinski/Anubis

https://github.com/danielkrupinski/GOESP

**Reply by ghost** [reply]
Ok ty very much
